### ISSUE TITLE
Final_report_review_jx266

### ISSUE THREAD

**Issue by jx266** [fjt37/ml-big-data-project#12]
The topic is bitcoin projection. It is a very popular topic at present for its unpredicted change and high potential value. The author provided two uses of prediction for the price of bitcoin including speculative investment and online purchase.

There are two main datasets. The first dataset contains many important features which are some historical direct data and indirect data. Then, this dataset is preprocessed to drop the first 1000 days’ data to better reflect the recent change of bitcoin price.  The second data set is about political events’ impact on bitcoin.  For data collection and cleaning parts, I think this project did pretty good. But I think you could give more details like if there is any missing data or unrelated features, and how you deal with that.

For preliminary model analysis, you have very brief and clear explanation on extracting important features, safe number of features to avoid overfitting and optimal days of historical data. But I am a little confused about how you decide 70 is the safe number because the trend of mean error is increasing at 70. So why not choosing less number? For main analysis, I hope to see the comparison plots of MSE for your different choices, such as l1 and l2 Huber loss functions. 

In general, it is a very good report and your final model has a great prediction on the test set. Hope you guys can keep improving your model for analyzing bitcoin.
